pom.xml

```diff
@@ -655,20 +526,6 @@
       <activation>
         <jdk>[1.8,1.9)</jdk>
       </activation>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-javadoc-plugin</artifactId>
-              <configuration>
-                <javadocVersion>1.8</javadocVersion>
-                <additionalparam>-Xdoclint:all,-Xdoclint:-missing</additionalparam>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
-      </build>
     </profile>
   </profiles>
-</project>
+</project>
```

Comment:
[trivial] This change deletes the trailing newline at the end of the file, which should remain. This happens with certain text editors.